lib/wifi_user/email_sender.rb

```diff
@@ -100,4 +100,8 @@ def self.send_active_users_signup_survey(user)
   def self.do_not_reply_email_address_id
     ENV["NOTIFY_DO_NOT_REPLY"] || "0d22d71f-afa3-4c72-8cd4-7716678dbd43"
   end
+
+  def self.support_reply_email_address_id
+    ENV["NOTIFY_SUPPORT_REPLY"] || "5619b95b-2f93-4a70-be64-2f2568f8876f"
```

Comment:
oh I don't think this UUID should be hidden here, could it be a constant or somehow easy to find? I'm sure it'll be different on each env?

Reply:
It's just a default so that if the variable isn't set we use that (which so happens to be the production reply-to-id) In practice we shouldn't really need it